### PR TITLE
ENH/DOC: Acknowledgements

### PR DESCRIPTION
**Pull request by aburrell**
# Description

Updates the funding acknowledgements to current standards.

# Type of change

Please delete options that are not relevant.

- New feature (non-breaking change which adds functionality)
- This change requires a documentation update

# How Has This Been Tested?

Ran flake8

## Test Configuration
* Operating system: OS X Big Sur
* Version number: Python 3.9
* Any details about your local setup that are relevant

# Checklist:

- [N/A] Make sure you are merging into the ``develop`` (not ``main``) branch
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] Add a note to ``CHANGELOG.md``, summarizing the changes

If this is a release PR, replace the first item of the above checklist with the
release checklist on the pysat wiki:
https://github.com/pysat/pysat/wiki/Checklist-for-Release

### CODE REVIEW

File: ACKNOWLEDGEMENTS.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+ - Cosmic Studio
+ - Defense Advanced Research Projects Agency (DARPA) Defense Sciences Office
+ - Office of Naval Research (ONR)
+ - National Aeronautics and Space Administration (NASA)
+ - National Science Foundation (NSF)
+
+Programs
+--------
+ - NSF 125908, AGS-1651393
+ - Naval Research Laboratory N00173191G016 and N0017322P0744
```
Comment (aburrell) [inline ACKNOWLEDGEMENTS.md:17]: @rstoneback @jklenzing @JonathonMSmith please check to ensure these funding sources are correct for this package.  Can confirm that NSF 125908, NRL/ONR/DARPA are correct.

Comment (jklenzing) [inline ACKNOWLEDGEMENTS.md:17]:
```suggestion
 - Naval Research Laboratory N00173191G016 and N0017322P0744
 - The Space Precipitation Impacts (SPI) project at Goddard Space Flight Center through the Heliophysics Internal Science Funding Model
```

File: ACKNOWLEDGEMENTS.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+The following institutions, missions, and programs have provided funding
+for pysatSpaceWeather development.
+
+Institutions
+------------
+ - Cosmic Studio
+ - Defense Advanced Research Projects Agency (DARPA) Defense Sciences Office
+ - Office of Naval Research (ONR)
+ - National Aeronautics and Space Administration (NASA)
+ - National Science Foundation (NSF)
```
Comment (jklenzing) [inline ACKNOWLEDGEMENTS.md:12]:
```suggestion
 - National Aeronautics and Space Administration (NASA)
 - National Science Foundation (NSF)
 - Office of Naval Research (ONR)
```

File: ACKNOWLEDGEMENTS.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+Funding
+=======
+The following institutions, missions, and programs have provided funding
+for pysatSpaceWeather development.
+
+Institutions
+------------
+ - Catholic University of America (CUA)
```
Comment (rstoneback) [inline ACKNOWLEDGEMENTS.md:8]:
```suggestion
 - The Catholic University of America (CUA)
```
I checked online and the name officially starts with 'The'. Technically it was the same thing at UTD. CUA should be placed at the end.

Comment (aburrell) [reply ACKNOWLEDGEMENTS.md:8]: "The" is ignored when alphabetizing.

**Review by jklenzing — APPROVED** [review]

File: ACKNOWLEDGEMENTS.md (hunk 3 above)
Comment (rstoneback) [reply ACKNOWLEDGEMENTS.md]: Depends upon the standard.

**Pull request merged**